core/src/main/java/io/questdb/griffin/engine/functions/bind/CompiledFilterSymbolBindVariable.java

```diff
@@ -99,6 +102,11 @@ public void toPlan(PlanSink sink) {
         sink.val("?::symbol");
     }
 
+    @Override
+    public void toSink(@NotNull CharSink<?> sink) {
+        sink.put('\'').put(getSymbol(null)).put('\'').put("::symbol");
```

Comment:
not sure if `getSymbol()` can return null?
if yes, we probably do not need the qoutes.